tests/CMakeLists.txt

```diff
@@ -481,6 +481,32 @@ add_test_case(mqtt311_listener_connection_events_no_session)
 add_test_case(mqtt311_listener_connection_events_with_session)
 add_test_case(mqtt311_listener_publish_event)
 
+# "rrsm" = "request_response_subscription_manager"
```

Comment:
Trivial: The `aws_rr_subscription_manager_init()` would immediately return an `AWS_ERROR_INVALID_ARGUMENT` error but should we still add a test that insures a max_subscriptions < 1 fails?